### PR TITLE
[WIP] Updated dockershim removal banner for release 1.24

### PR DESCRIPTION
**Pull request by chrisnegus**
I updated the wording on the dockershim removal banner to go into effect on Kubernetes 1.24 release day.

I marked it as WIP so people can review the wording and let me know if this should go into the dev-1.24 branch or just go into main after the release goes live.

### CODE REVIEW

**Comment by sftim** [comment]
Targeting _main_ would be OK with me. We can get this changed approved, with a hold, so that it is ready to unhold once the docs repo thaws post-release.

However, targeting _dev-1.24_ will be mean less work on the release day, and so I recommend that approach.

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -34,9 +34,9 @@ announcements:
   startTime: 2022-04-07T00:00:00 # Added in PR: https://github.com/kubernetes/website/pull/32805
   endTime: 2022-05-10T00:00:00 # Expires 2022-05-10, but will change text slightly on release date
   style: "background: #326ce5"
-  title: "Dockershim removal set for Kubernetes 1.24"
+  title: "Dockershim removed from Kubernetes 1.24"
```
Comment (sftim) [inline data/announcements/scheduled.yaml:37]:
```suggestion
  title: "Dockershim removed from Kubernetes"
```
(To me) the following paragraph makes the context clear enough. We also link to a frequently-asked-questions page.

Comment (chrisnegus) [reply data/announcements/scheduled.yaml:37]: Fair enough. I've made the change you suggested.

**Comment by chrisnegus** [comment]
> Targeting _main_ would be OK with me. We can get this changed approved, with a hold, so that it is ready to unhold once the docs repo thaws post-release.
> 
> However, targeting _dev-1.24_ will be mean less work on the release day, and so I recommend that approach.

Okay. I change the target branch to dev-1.24. A whole mess of 1.24 commits showed up on this PR. @sftim, please let me know if I did this wrong.

**Comment by sftim** [comment]
To move this work forward @chrisnegus, rebase against _dev-1.24_ and keep only commit 8b42653c35d4237f150ae4782bcea748f949f6b7 and onwards.

**Comment by chrisnegus** [comment]
I was having some trouble with the rebasing, so I opened [PR 33229](https://github.com/kubernetes/website/pull/33229) to replace this.

**Pull request closed without merging**